subsys/bluetooth/controller/Kconfig

```
@@ -250,8 +250,12 @@ config BT_CTLR_ISO_VENDOR_DATA_PATH
 choice BT_CTLR_TX_PWR
 	prompt "Tx Power"
 	default BT_CTLR_TX_PWR_0
+	depends on BT_CTLR_TX_PWR_ANTENNA=0 || BT_LL_SW_SPLIT
```

Not sure of the syntax here, `=` or `==` , otherwise missing whitespaces
```suggestion
	depends on BT_CTLR_TX_PWR_ANTENNA = 0 || BT_LL_SW_SPLIT
```